Classes/Controllers/PBGitCommitController.m

```diff
@@ -187,7 +185,12 @@ - (void) commitWithVerification:(BOOL) doVerify
 	self.isBusy = YES;
 	[commitMessageView setEditable:NO];
 
-	[index commitWithMessage:commitMessage andVerify:doVerify];
+	[repository.index commitWithMessage:commitMessage andVerify:doVerify];
+}
+
+
+- (PBGitIndex *) index {
```

Comment:
It is used from PBGitIndexController and PBWebChangesController, so those would need reworking to not repeat that chain of accessors over and over.

I didn’t want to get into that now, because it’s not really clear to me what the relations between those controllers and the objects they manipulate _should_ be.